examples/llava/clip-impl.h

```diff
@@ -96,9 +89,9 @@ enum projector_type {
     PROJECTOR_TYPE_MLP_NORM,
     PROJECTOR_TYPE_LDP,
     PROJECTOR_TYPE_LDPV2,
-    PROJECTOR_TYPE_RESAMPLER,
+    PROJECTOR_TYPE_MINICPMV,
     PROJECTOR_TYPE_GLM_EDGE,
-    PROJECTOR_TYPE_MERGER,
+    PROJECTOR_TYPE_QWEN2VL,
```

Comment:
cc @HimariO , `PROJECTOR_TYPE_RESAMPLER` is renamed to `PROJECTOR_TYPE_QWEN2VL`

For qwen2.5, we can add `PROJECTOR_TYPE_QWEN25VL`. For code paths used by qwenvl, we will need to check `ctx->proj_type == PROJECTOR_TYPE_QWEN2VL || ctx->proj_type == PROJECTOR_TYPE_QWEN25VL`

But tbh the best way is to have a dedicated builder function for qwenvl, it makes the code mush easier to read. I'll make a proposal in the next few days.